client/app/dedicatedCloud/license/enable/dedicatedCloud-license-enable-us.controller.js

```diff
@@ -0,0 +1,69 @@
+angular.module("App").controller("DedicatedCloudLicencesSplaEnableUSCtrl", class DedicatedCloudLicencesSplaEnableUSCtrl {
+
+    constructor ($stateParams, $scope, $state, OvhHttp, User, constants) {
+        if (constants.target !== "US") {
```

Comment:
Why do you make the test in controller instead of in route declaration?
```
if (constants.target === "US") {
    $stateProvider.state(..., { ... });
}
```